src/dotnet/UI.Blazor.App/Components/NoResults/NoResults.razor

```diff
@@ -0,0 +1,11 @@
+@namespace ActualChat.UI.Blazor.App.Components
```

Comment:
Why did you put here? Not next to FoundChatList?

Reply:
It's just a special cat component, so I decided to put it along with other cat components